database.py

```
@@ -0,0 +1,10 @@
+import redis
+import os
+
+REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
+
+try:
+    redis_conn = redis.from_url(REDIS_URL, decode_responses=True)
+except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)